app/views/customers/show.html.erb

```diff
@@ -20,6 +20,7 @@
           <div class="col-sm-2">
             <div id="profile-date">
               <small>Subscriber since <%= @customer.created_at.strftime "%B %Y" %></small>
+              <%= link_to "Click here to view invoice", invoice_path %>
```

Comment:
Consider moving to the dropdown menu for the customer navbar in views/application/_navbar_customer.html.erb?